src/screens/AllAddresses.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag'
+import { WALLET_HISTORY_COUNT } from '../constants';
+import helpers from '../utils/helpers';
+import path from 'path';
+import AddressList from '../components/AddressList';
+
+/**
+ * Screen that has a list of addresses of the wallet
+ *
+ * @memberof Screens
+ */
+class AllAddresses extends React.Component {
```

Comment:
suggestion(non-blocking): I'd suggest that all new screens and components should already be written in functional component style, to push forward our migration away from class components.

Reply:
This actually should be blocking. Refactored to a functional component. https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/8b18b331c1d863c570559aadf9fbcb1efad03eac